### PR TITLE
Adding MKR NB 1500 as built target for CI

### PR DESCRIPTION
**Pull request by aentinger**
This pull requests adds the `MKR NB 1500` as build target for the Travis based CI process.

### CODE REVIEW

File: examples/ArduinoIoTCloud_Travis_CI/thingProperties.h [hunk 1]
```diff
@@ -8,8 +8,9 @@
 #if defined(BOARD_HAS_WIFI)
 #elif defined(BOARD_HAS_GSM)
 #elif defined(BOARD_HAS_LORA)
+#elif defined(BOARD_HAS_NB)
 #else
-  #error "Arduino IoT Cloud currently only supports MKR1000, MKR WiFi 1010, MKR GSM 1400 and MKR WAN 1300/1310"
```
Comment (per1234) [inline examples/ArduinoIoTCloud_Travis_CI/thingProperties.h]: Was it intentional to remove MKR WAN 1300/1310 from the error message?

Comment (aentinger) [reply examples/ArduinoIoTCloud_Travis_CI/thingProperties.h]: They were not removed but moved towards the end ;)

Comment (per1234) [reply examples/ArduinoIoTCloud_Travis_CI/thingProperties.h]: I still don't see it:
https://github.com/arduino-libraries/ArduinoIoTCloud/blob/138f726a30e042d2b41df764e7e97ff240099eb3/examples/ArduinoIoTCloud_Travis_CI/thingProperties.h#L13
but I notice now this is just the CI sketch, so not really important either way.

Comment (aentinger) [reply examples/ArduinoIoTCloud_Travis_CI/thingProperties.h]: Oh my, you are right. Fixed it. Interestingly enough I clearly remember preserving those changes ...

**Pull request merged**